packages/stark-ui/src/modules/toast-notification/components/toast-notification.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="stark-toast" [ngClass]="getMessageTypeClass()">
+
+	<mat-icon svgIcon="information" *ngIf="message.type === 0"></mat-icon>
```

Comment:
Could you please add `starkSvgViewBox` directive in mat-icon ? 
This allows to resize easily the icon.